dspace-api/src/test/data/dspaceFolder/config/spring/api/external-services.xml

```diff
@@ -91,4 +91,16 @@
         </property>
     </bean>
 
-</beans>
\ No newline at end of file
+    <bean id="dataciteLiveImportDataProvider" class="org.dspace.external.provider.impl.LiveImportDataProvider">
```

Comment:
This same bean needs to be added to the default `dspace/config/spring/api/external-services.xml` file.  Without it, I couldn't get this PR to work correctly.